src/api/app/views/webui2/shared/_repositories_flag_table_column.html.haml

```diff
@@ -0,0 +1,9 @@
+:ruby
+  title = flag.effective_status == 'disable' ? 'Disabled' : 'Enabled'
+  title += ' (default)' if flag.new_record?
+  content = String.new(render(partial: 'shared/flag_popover', locals: { flag: flag, project: project, package: package, remote: !flag.has_children }))
```

Comment:
Can you please describe again what was the issue here?

Reply:
When we did

`content = render(partial: 'shared/flag_popover', locals: { flag: flag, project: project, package: package, remote: !flag.has_children })`

and passed `content` as data attribute for popover, the view was broken. We realized that `content` was not `String` but `ActiveSupport::SafeBuffer`. So the `data` value received something different from a `String`.

Neither Haml nor `ActiveSupport::SafeBuffer#to_s` convert it to String. We had two options to do so: `String.new(...)` or interpolation (which was an offense).

Reply:
Thanks for the explanation.

Reply:
offenses can be disabled. which everybody from the future will understand. this, no one will understand.